_includes/comment-form.html

```diff
@@ -0,0 +1,96 @@
+        }
+        
+        function onSubmit(token) {
+            var form = document.getElementById('feedback-form');
+            if(form.checkValidity()){
+                form.submit();
+            }
+        }
+    </script>
+    <script>
+        function toggleFeedbackForm() {
+            var commentDiv = document.getElementById("comment-form-div");
+            var toggleButton = document.getElementById("comment-form-toggle-button");
+
+            if (commentDiv.style.display === "none") {
+                $(commentDiv).slideDown("slow");
+                toggleButton.style.display = "none";
+            } else {
+                $(commentDiv).slideUp("slow");
+                toggleButton.style.display = "block";
+            }
+        }
+    </script>
+    {% endif %}
+    <div class="widget">
+        <button id="comment-form-toggle-button" class="newsletter-button" onclick="toggleFeedbackForm()">Scrivimi cosa ne pensi</button>
+        <div id="comment-form-div" class="widget" style="display: none">
+            <div class="widget__head">
+                <h4 class="widget__title">Scrivimi cosa ne pensi</h4>
+            </div>
+            <form id="feedback-form" action="{{ site.get-form-action-url }}" method="POST">
+            <div class="comments-group">
+                <div class="comments-group-top">
+                    <input type="hidden" name="article" value="{{page.title}}">
+                    
+                    <div onfocusout="onFocusOut(this)">
+                        <input id="name" class="comments-item" type="text" name="name" placeholder="Nome" required/>
+                    </div>
+                    
+                    <div onfocusout="onFocusOut(this)">
+                        <input  class="email comments-item" type="email" name="email" placeholder="Email Address" required/>
+                    </div>
+            
+                    <div onfocusout="onFocusOut(this)">
+                        <textarea class="required comments-item" style="height: 100px;" rows="4" cols="50" name="feedback" placeholder="Scrivi un commento" required></textarea>
+                    </div>
+
+                    <div class="inline-content">
+                        <div>
+                            <label class="required comments-item" for="rating">Ti e' piaciuto?</label>
+                            <fieldset class="rating comments-item">
+                                <input type="radio" id="star5" name="rating" value="5" required/><label class = "full" for="star5" title="Awesome - 5 stars"></label>
+                                <input type="radio" id="star4half" name="rating" value="4 and a half" required/><label class="half" for="star4half" title="Pretty good - 4.5 stars"></label>
+                                <input type="radio" id="star4" name="rating" value="4" required/><label class = "full" for="star4" title="Pretty good - 4 stars"></label>
```

Comment:
Bisogna tradurre i title

Reply:
Proposta:

0.5-1.5 -> Fa schifo
2-2.5  -> Non mi è piaciuto
3-3.5 -> Niente male
4-4.5 -> interessante
5 -> fantastico

Reply:
Dobbiamo trovare qualcosa di piu' polite per `0.5-1.5 -> Fa schifo` :)

Che ne dici di `0.5-1.5 -> Terribile`

Reply:
si in effetti è un po' too much ahah
Terribile mi piace